packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -84,22 +91,38 @@ const getNewValue = (
 	return [ value ];
 };
 
+function computeCompositeItemId(
+	prefix: string,
+	filterElement: NormalizedFilter[ 'elements' ][ number ]
+) {
+	return `${ prefix }-${ filterElement.value }`;
+}
+
 function ListBox( { view, filter, onChangeView }: SearchWidgetProps ) {
-	const compositeStore = useCompositeStore( {
-		virtualFocus: true,
-		focusLoop: true,
+	const baseId = useInstanceId( ListBox, 'dataviews-filter-list-box' );
+
+	const [ activeCompositeId, setActiveCompositeId ] = useState<
+		string | null | undefined
+	>(
 		// When we have no or just one operator, we can set the first item as active.
-		// We do that by passing `undefined` to `defaultActiveId`. Otherwise, we set it to `null`,
-		// so the first item is not selected, since the focus is on the operators control.
-		defaultActiveId: filter.operators?.length === 1 ? undefined : null,
-	} );
+		// We do that by setting the initial `activeId` to `undefined`.Otherwise,
+		// we set it to `null`, so the first item is not selected,
+		// since the focus is on the operators control.
+		filter.operators?.length === 1 ? undefined : null
+	);
+	const firstCompositeElementRef = useRef< HTMLElement | null >( null );
```

Comment:
It can be deleted. Initially I was calling `.focus()` on it, but then I switched to generating custom composite IDs , which allows to call 	`setActiveItemID` without imperatively calling `firstCompositeElementRef.focus()`. I will clean up in the dedicated PR